src/components/tree-item/tree-item.tsx

```diff
@@ -311,22 +311,25 @@ export class TreeItem
         role="treeitem"
       >
         <div class={{ [CSS.itemExpanded]: isExpanded }}>
-          <div
-            class={{
-              [CSS.nodeContainer]: true,
-              [CSS_UTILITY.rtl]: rtl
-            }}
-            data-selection-mode={this.selectionMode}
-            ref={(el) => (this.defaultSlotWrapper = el as HTMLElement)}
-          >
-            {chevron}
-            {itemIndicator}
-            {this.iconStart ? iconStartEl : null}
-            {checkbox ? checkbox : defaultSlotNode}
+          <div class={CSS.nodeAndActionsContainer}>
```

Comment:
Would it be possible to merge this with the above container node?

Reply:
Per our conversation, disregard this one. I didn't notice `nodeAndActionsContainer` didn't contain all of the expanded node content.